.github/ISSUE_TEMPLATE/ad-hoc-meeting.yaml

```diff
@@ -0,0 +1,53 @@
+name: 'Create an Ad-Hoc Meeting'
+description: 'This Issue template is used for creating a new Ad-Hoc Meeting for the Cross Project Council'
+title: "[Ad hoc meeting] "
+assignees:
+  - bensternthal
+labels:
+  - cross-project-council-agenda
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Agenda for the Meeting
+  - type: input
+    id: facilitator
+    attributes:
+      label: Facilitator
+      description: Who is the facilitator of this meeting?
+    validations:
+      required: true
+  - type: input
+    id: related-issues
+    attributes:
+      label: Related Issue(s)
+      description: What are the related Issues to this Meeting?
+    validations:
+      required: false
+  - type: input
+    id: meeting-date
+    attributes:
+      label: Meeting Date
+      description: 'When should the meeting happen? Use a "yyyy-MM-dd HH:mm SSZ" format (e.g.: 2009-05-27 UTC)'
+    validations:
+      required: true
+  - type: input
+    id: zoom-link
+    attributes:
+      label: Zoom/G Meet Libk
+      description: Where should the meet happen?
+      value: 'https://zoom.us/j/91398697172?pwd=QVgxRUFXdmtHVUNiODhoSU11ejF0QT09'
```

Comment:
Is this a default link that works all the time or does it need to be updated later? It would be good to clarify.

Reply:
I sadly have no idea. Maybe @kyliewd or @bensternthal know about that? (I'm not a Zoom pro 😅)

Reply:
@ovflowd I would suggest that the zoom link be left as blank or removed. The problem with using the same zoom link all the time.. is that it might already be in use at that time :)

Practically, I see the zoom link being generated when the calendar event is created.